wai-aria/role/synonym-roles.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
+<head>
+  <title>Region Role Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<!-- spec resolution https://github.com/w3c/core-aam/issues/166 -->
+
+<div role="none" id="none" data-expectedrole="none" data-testname="none role == computedrole none" class="ex">x</div><!-- preferred -->
+<div role="presentation" id="presentation" data-expectedrole="none" data-testname="synonym presentation role == computedrole none" class="ex">x</div><!-- synonym -->
+
+<div role="image" id="image" data-expectedrole="image" data-testname="image role == computedrole image" class="ex">x</div><!-- preferred -->
+<div role="img" id="img" data-expectedrole="image" data-testname="synonym img role == computedrole image" class="ex">x</div><!-- synonym -->
+
+<!-- `directory` synonym deprecated in ARIA 1.2; these examples should all return computedrole `list` -->
+<ul id="list1" data-expectedrole="list" data-testname="ul == computedrole list" class="ex"><li>x</li></ul>
```

Comment:
you aren't including html implicit roles anywhere else, so why here?

Reply:
I am in /wai-aria/role/basic.html, and I added this here because of Simon's review comment in https://github.com/web-platform-tests/wpt/pull/38925/files#r1145967470

Reply:
Unless you're suggesting we remove this 1 line but leave the trailing 3. If so, np I will delete it.

Reply:
```suggestion
```